pom.xml

```diff
@@ -382,6 +382,17 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
```

Comment:
Our previous concern was that the download source was flakiness: https://github.com/apache/spark/pull/49710#discussion_r1932354650

If we can confirm that there's no issue with it, that would be great.